src/main/java/swift/ui/MainWindow.java

```diff
@@ -170,10 +175,45 @@ private void handleExit() {
         primaryStage.hide();
     }
 
+    /**
+     * Switch to tasks tab.
+     */
+    @FXML
+    private void handleTaskTab() {
+        logic.showAllLists();
+
+        personListPanel.removeHeading();
+        taskListPanel = new TaskListPanel(logic.getFilteredTaskList(), logic.getFilteredBridgeList(),
+                logic.getFilteredPersonList());
+        listPanelPlaceholder.getChildren().add(taskListPanel.getRoot());
+
+        personTaskListPanel.removeHeading();
+        taskPersonListPanel = new TaskPersonListPanel(logic.getFilteredPersonList());
```

Comment:
You might want to show nothing in the TaskPersonListPanel when you first switch because it doesn't really make sense to show all contacts under the Associated Contacts header